utils/truncate.ts

```
@@ -0,0 +1,16 @@
+export const truncateAddress = (fullStr: `0x${string}`, strLen: number) => {
+  if (fullStr.length <= strLen) return fullStr
+
+  const separator = '...'
+
+  var sepLen = separator.length,
+    charsToShow = strLen - sepLen,
+    frontChars = Math.ceil(charsToShow / 2),
+    backChars = Math.floor(charsToShow / 2)
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)